### ISSUE TITLE
cooperating voting systems in parallel

### ISSUE THREAD

**Issue by jean-raphael** [tournesol-app/tournesol#944]
Add the possibility to have several voting systems working together.

For example for youtube, we could have a voting system by video (as currently) but also by channel, with a vote comparing the different channels.
But also with a voting system for music videos which are completely rejected by the current system

**Comment by GresilleSiffle** [comment]
Hell @jean-raphael 

Thank you for taking the take to give us feedback :+1:

As a Tournesol developer I'm going to give you my **personal opinion**. I'm speaking for myself so don't take it as the official Tournesol team's point of view.

> Add the possibility to have several voting systems working together.

> For example for youtube, we could have a voting system by video (as currently) but also by channel, with a vote comparing the different channels.

The first question I ask myself when I read this is: why? If the motivation is not clear, the purpose of such a system in term of public interest won''t be clear too.

Currently we don't plan to add new compare-able entities (like channels) in short or medium term. The first reason is because we are focused on the UX of the existing videos poll, but comparing channels is something we are thinking about.

The second question is what is a recommended channel? and what is a non-recommended channel? Is it a channel with a lot of well rated videos on Tournesol? and what is "well rated"? a high score? a high average score? a high median score?

If a recommended channel is a channel with well rated videos, we can already display such a list in Tournesol. We don't need to ask the users to compare channels if they have already compared their videos. If a recommended channel is not deductible from the current data we have, we might need to create a dedicated poll.

This might be a delicate operation as the results of the channels poll may not be coherent with the results of the videos poll.

I personally don't like that much the idea of comparing channels, or even creating a ranking based on the Tournesol data:

a ~ I think it will hurt a lot the diversity of the recommended information. Currently the top videos of Tournesol come from different channels and are supposedly recommended regardless of their author. This is because Tournesol focus on the information rather than on its origin.

b ~ I think it can shadow a lot information of public interest. Some YouTube channels are focused on a specific topic, and not all videos are video of public interest, but some of them are. What if a YouTube channel has hundred if non-recommended videos on Tournesol, but has the one that is the most recommended on Tournesol? Depending on how we compute the score of this channel, we may shadow this great video.

c ~ Videos are information, channels are groups of videos of different quality. Recommending videos to people may create a better world, a world where people will take better decisions. Recommending channels to people is in my opinion far less reliable.

d ~ Comparing videos is sometimes difficult, comparing channels might be even more.

To sum up, I'm more in favor in displaying an horizontal ordered list of channels, on several lines, based on the current Tournesol data (because it's fun), rather than displaying a vertical ranking of channels in a dedicated poll (because it will hurt the diversity of information).

**Comment by jean-raphael** [comment]
ok thanks

by rejected i mean,
because music is never fundamentally important, the voting system encourages people to vote the against any music video.
That's why I imagined a parallel sunflower with a music rating


> this is currently our definition of "good video".

ok i understand the concept but it's far from my intuitive definition of a good video,

In fact, if I understand correctly, this system emits statistically as many positive and negative notes which is quite arbitrary and can be representative of the qualities and defects of the videos.

And this average of the notes to zero seems to me a little far from the idea that I have of a good video

but ok i get it
